open_gpt/models/llama/loading.py

```diff
@@ -73,4 +74,25 @@ def load_model_and_tokenizer(
         trust_remote_code=True,
     )
 
+    """
+    Use LORA for inference, `peft_model_id_or_path` can be either an id on huggingface 
+    or the name of a directory. If it's a directory, there should be two files in it:
+    checkpoint file which contains the weight of LORA, named `adapter_model.bin`,
+    config file which will be used to initialize LORA, named `adapter_config.json`.
+    ** The file names cannot be changed **
+    
+    For details about PeftModel.from_pretrained, see:
+    `peft.utils.peft_model.py::load_adapter()` for loading ckpt,
+    `peft.utils.config.py::from_pretrained()` for loading config.
```

Comment:
Rephrase the docstr for this API with more details. It is weird just to mention lora here.